src/components/tx-flow/flows/AddOwner/styles.module.css

```diff
@@ -13,3 +13,11 @@
   align-self: center;
   text-transform: uppercase;
 }
+
+.bgDefault {
+  background-color: var(--color-background-main);
+}
+
+.bgInfo {
+  background-color: var(--color-info-background);
+}
```

Comment:
Nit: creating a class is excessive for one property. I'd suggest accessing the palette from the `sx` prop instead.